Source/JavaScriptCore/dfg/DFGOperations.cpp

```diff
@@ -3828,6 +3828,10 @@ JSC_DEFINE_JIT_OPERATION(operationGetDynamicVar, EncodedJSValue, (JSGlobalObject
     Identifier ident = Identifier::fromUid(vm, impl);
     RELEASE_AND_RETURN(throwScope, JSValue::encode(scope->getPropertySlot(globalObject, ident, [&] (bool found, PropertySlot& slot) -> JSValue {
         if (!found) {
+#if USE(BUN_JSC_ADDITIONS)
+            if (ident.isPrivateName() || ident.isWellKnownSymbol(vm))
+                return identifierToJSValue(vm, ident);
+#endif
```

Comment:
☝️ Allows passing `@@iterator` and other symbols to methods like `foo.@__lookupGetter(@@iterator)`.